src/web/templates/admin/prizes/prize_form_modal.html

```diff
@@ -50,13 +51,23 @@ <h4 class="mb-3">
                 {% endif %}
             >
                 <div class="row mb-3">
-                    {{ admin_macros.number_input(
-                        name='value',
-                        label=_('%(string)s:', string=_('Value')),
-                        help_text='',
-                        step=0.01,
-                        data=data, errors=errors
-                    ) }}
+                    {% if action == 'create' %}
+                        {{ admin_macros.text_input(
+                            name='values',
+                            label=_('%(string)s:', string=_('Values')),
+                            placeholder=_('E.g.: %(string)s', string='1000 500,5 333.33'),
+                            help_text=_('List of values of the prizes to create. Values can be floats and have to be positive.'),
```

Comment:
You have to say that they have to be space-separated, otherwise it's gonna be confusing